src/_guides/libraries/writing-package-pages.md

```diff
@@ -0,0 +1,335 @@
+shows that your package is worth trying.
+
+{{ site.alert.note }}
+  The package README is used in multiple ways.
+  For example, its content appears not only in the package page on pub.dev,
+  but also in [dartdoc][]-produced API reference documentation.
+{{ site.alert.end }}
+
+Although this page features the [`in_app_purchase`][] package README,
+yours might not need to be as large or detailed.
+If your package is simple and has no associated UI,
+its README might look more like the one for the [`yaml`][] package. 
+
+
+## Seven tips for good README
+
+Here are some suggestions for creating a README
+that works well on pub.dev:
+
+1. [Put a short description at the top](#tip1)
+2. [Include visual content](#tip2)
+3. [Use lists to present key information](#tip3)
+4. [Include usage examples](#tip4)
+5. [Use Dart code formatting](#tip5)
+6. [Mention related terms](#tip6)
+7. [Tell users where to go next](#tip7)
+
+
+### 1. Put a short description at the top {#tip1}
+
+According to our user research,
+package users spend only a few seconds to
+read the package description and decide whether to
+read the rest of the README.
+Thus, you should concisely describe what the package does or achieves,
+at a glance.
+Spend some time to craft a short and sweet description and
+help the user to make decisions.
+
+{{site.alert.tip}}
+  Don't write the package name again at the top.
+  It's already visible in the pub.dev UI.
+{{site.alert.end}}
+
+Here are some examples of good descriptions:
+
+*   `A Flutter plugin for showing rainbows.`
+*   `Use machine learning to categorize bird sounds.`
+
+Important information such as project status or constraints
+should also be near the top.
+For example:
+
+*   `Does not work on iOS versions below 10.3.`
+
+Here's a screenshot of the [`in_app_purchase`][] package page,
+which starts with a brief explanation of the package and a caution.
+
+{% asset libraries/package-page-description.png alt="description of the package in_app_purchase" class="screenshot" %}
+
+[Badges][] are often near the top of the README,
+either above or under the short description. 
+
+
+### 2. Include visual content {#tip2}
+
+If your package page is a wall of text with no visual content,
+users might find it intimidating and stop reading.
+Images are especially important if your package supports UI,
+but they're also useful for explaining important concepts.
+Either way, visual content can help users
+feel confident about using the package.
+
+Place visual content such as static images, animated GIFs, and
+videos (such as MOV or MP4 files)
+close to the beginning of the README,
+where users are likely to see them.
+
+{{site.alert.tip}}
+  Prefer animated GIFs and videos for UI-related content,
+  because most UIs aren't static,
+  and animation conveys more information about the UI's behavior.
+{{site.alert.end}}
+
+The screenshots below show how
+adding visual content made the `in_app_purchase` package page look better.
```

Comment:
Maybe instead of saying it looks `better`, we could be a bit more specific. Something more targetted such as `more informative`, `more fleshed out`, `more engaging`, or similar (I'm not really sure) to better indicate how it is "better".

Reply:
What about "look informative at first glance"? :)